includes/Handlers/Connection.php

```diff
@@ -343,13 +343,12 @@ public function handle_disconnect() {
 
 			$this->disconnect();
 
-			facebook_for_woocommerce()->get_message_handler()->add_message( __( 'Uninstall successful', 'facebook-for-woocommerce' ) );
+			facebook_for_woocommerce()->get_message_handler()->add_message( __( 'Disconnect successful', 'facebook-for-woocommerce' ) );
```

Comment:
I have updated the messages but any suggestions are still welcome @j111q ( I just wanted to push the PR ).

Reply:
+1 to @haszari's comments, and actually I think I would align these two message: `Disconnection successful.` and `Connection successful!` 

I'd remove `Thanks for using...` just because that's not a pattern I've seen elsewhere, and though it's a nice statement, I think we don't really thank users for using a feature 😅